### PR TITLE
CORE-10453 : Interfaces for alias identity look up

### PR DESCRIPTION
**Pull request by SaurabhR3**
Added interfaces for alias identity look up

For more details : https://r3-cev.atlassian.net/wiki/spaces/CB/pages/4448321990/WIP+Alias+Identities+at+Flow+Level

### CODE REVIEW

File: interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.interop;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+@CordaSerializable
+public interface AliasMemberInfo {
```
Comment (yarivus) [inline interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java]:
```suggestion
import net.corda.v5.base.annotations.CordaSerializable;
import org.jetbrains.annotations.NotNull;
import java.util.List;

@CordaSerializable
public interface AliasMemberInfo {
```

File: interop/src/main/java/net/corda/v5/interop/HoldingIdAliasGroupInfo.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package net.corda.v5.interop;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+
+import java.util.List;
```
Comment (yarivus) [inline interop/src/main/java/net/corda/v5/interop/HoldingIdAliasGroupInfo.java]:
```suggestion
import net.corda.v5.base.annotations.CordaSerializable;
import java.util.List;
```

**Review by yarivus — CHANGES_REQUESTED** [review]

File: interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java (hunk 1 above)
Comment (SaurabhR3) [reply interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java]: Fixed in 730895f

File: interop/src/main/java/net/corda/v5/interop/HoldingIdAliasGroupInfo.java (hunk 2 above)
Comment (SaurabhR3) [reply interop/src/main/java/net/corda/v5/interop/HoldingIdAliasGroupInfo.java]: Fixed in 730895f

File: interop/src/test/java/net/corda/v5/interop/HoldingIdAliasGroupInfoTest.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class HoldingIdAliasGroupInfoTest {
+
+    private final HoldingIdAliasGroupInfo holdingIdAliasGroupInfo = mock(HoldingIdAliasGroupInfo.class);
+
+    @Test
+    void getShortHash() {
```
Comment (szymonsztuka) [inline interop/src/test/java/net/corda/v5/interop/HoldingIdAliasGroupInfoTest.java:18]: Those tests test Mockito framework, the don't test the code business logic, please remove the whole class.

File: interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.interop;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+import org.jetbrains.annotations.NotNull;
+import java.util.List;
+
+@CordaSerializable
+public interface AliasMemberInfo {
```
Comment (szymonsztuka) [inline interop/src/main/java/net/corda/v5/interop/AliasMemberInfo.java:8]: None of the interfaces have any JavaDoc !

**Review by yarivus — APPROVED** [review]
LGTM

**Review by szymonsztuka — CHANGES_REQUESTED** [review]
This PR anyway needs to wait before merging for a matching PR in `corda-runtime-os`  using these interfaces (I mean they will be reviewed together).

**Pull request closed without merging**